library/sonarqube

```diff
@@ -4,16 +4,16 @@ Maintainers: Janos Gyerik <janos.gyerik@sonarsource.com> (@janos-ss),
              Pierre Guillot <pierre.guillot@sonarsource.com> (@pierre-guillot-sonarsource),
              Michal Duda <michal.duda@sonarsource.com> (@michal-duda-sonarsource)
 GitRepo: https://github.com/SonarSource/docker-sonarqube.git
-GitCommit: 2f7290e97a56f71603eff494db0d6b8d34ca426e
+GitCommit: 2618ed9a5371c30736de1e0d5dbcc9e7aa05f105
 
 Tags: 7.9.3-community, 7.9-community, lts
 Directory: 7/community
 
-Tags: 8.3.1-community, 8.3-community, 8-community, community, latest
+Tags: 8.4.0-community, 8.3.1-community, 8.3-community, 8-community, community, latest
```

Comment:
I don't think this is quite what you intended -- this includes all of `8.4.0`, `8.3.1`, and `8.3` as tags for the same image.

Reply:
this was indeed not the plan. Thanks for the catch (: 
corrected in my latest commit